packages/types/src/discord/guild.ts

```diff
@@ -3,33 +3,67 @@
 import type { ChannelTypes, DiscordChannel, DiscordOverwrite, DiscordThreadMember, SortOrderTypes } from './channel.js'
 import type { DiscordEmoji } from './emoji.js'
 import type { DiscordGuildCreateExtra } from './gateway.js'
+import type { DiscordInvite } from './invite.js'
 import type { OAuth2Scope } from './oauth2.js'
 import type { DiscordRole } from './permissions.js'
 import type { DiscordSticker } from './sticker.js'
 import type { DiscordAvatarDecorationData, DiscordUser } from './user.js'
 
 /** https://discord.com/developers/docs/resources/guild#guild-object-guild-structure */
 export interface DiscordGuild extends Partial<DiscordGuildCreateExtra> {
+  /** Guild id */
+  id: string
   /** Guild name (2-100 characters, excluding trailing and leading whitespace) */
   name: string
+  /** Icon hash */
+  icon: string | null
+  /** Icon hash, returned when in the template object */
+  icon_hash?: string | null
+  /** Splash hash */
+  splash: string | null
+  /** Discovery splash hash; only present for guilds with the "DISCOVERABLE" feature */
+  discovery_splash: string | null
   /** True if the user is the owner of the guild */
   owner?: boolean
+  /** Id of the owner */
+  owner_id: string
+  /** Total permissions for the user in the guild (excludes overwrites and implicit permissions) */
+  permissions?: string
```

Comment:
https://discord.com/developers/docs/resources/guild#guild-object-guild-structure

> These fields are only sent when using the [GET Current User Guilds](https://discord.com/developers/docs/resources/user#get-current-user-guilds) endpoint and are relative to the requested user

Should we add this note?


```suggestion
  /** 
   * Total permissions for the user in the guild (excludes overwrites and implicit permissions)
   * @remarks This field is only sent when using the {@link getGuilds | getGuilds} helper
   */
  permissions?: string
```